pandas/io/parsers.py

```diff
@@ -320,26 +329,33 @@
     tool, ``csv.Sniffer``. In addition, separators longer than 1 character and
     different from ``'\s+'`` will be interpreted as regular expressions and
     will also force the use of the Python parsing engine. Note that regex
-    delimiters are prone to ignoring quoted data. Regex example: ``'\r\t'``
+    delimiters are prone to ignoring quoted data. Regex example: ``'\r\t'``.
 delimiter : str, default ``None``
-    Alternative argument name for sep."""
+    Alias for sep.
+    """
+
+_see_also = ("to_csv : Write DataFrame to "
+             "a comma-separated values (csv) file.")
```

Comment:
If I'm not missing something, in the 3 functions reusing `_parser_params` this same `_see_also` is used.

Can you check if that's the case, and if it is move the content inside `_parser_params`? Also, it'd be nice to have the 3 functions self-reference themselves, and also the equivalent `to_*` functions.

Reply:
I add more in `See also` section to cross-reference each other except `read_table` due to being deprecated in 0.24.0.  I don't think we have `to_table` or `to_fwf` other than `to_csv`.